docs: add deploy post (#668)
devopness/devopness: docs/applications/deploy-application.md

---
title: Deploy application

> intro: Deploy applications to enable your users to interact with your application.

Reviewer: ```suggestion
intro: Deploy your code changes in smaller chunks to make it easier to validate the code through build and testing pipelines, reducing the feedback loop on error detection. Learn how to use Devopness to deploy application changes confidently, with zero downtime, using your own custom build pipelines and make sure your users can benefit from your application as soon as possible. The more often you deploy your application, the better it tends to improve.
```

    related:
    featured:
---

1. On Devopness, navigate to a project then select an environment
1. Find the `Applications` card
1. Click `View` in the `Applications` card to see a list of existing `Applications`
1. Click `DEPLOY` on the application you want to deploy

> 1. Choose branch and pipeline, to be used to deploy, then click `Next`

Reviewer: Is this comma really needed?
```suggestion
1. Choose the branch and the pipeline to be deployed, then click `Next`
```

1. Review deploy information then click `Confirm`

>     > There should be a green notification popup saying "Deployment triggered, please wait"

Reviewer: My suggestion is to avoid mentioning (as much as possible) some details that could change very often and would lead the documentation to become deprecated:
```suggestion
   > A notification popup will be displayed, confirming that the deployment has been triggered
```
